src/components/modal/v2/parts/ContentWrapper.jsx

```diff
@@ -19,11 +19,13 @@ const ContentWrapper = () => {
         product = useServerData().views[0].meta.product;
     }
 
-    const { headline, subheadline } = useContent(product);
+    const { headline, subheadline, qualifyingSubheadline } = useContent(product);
+    const { qualifying: isQualifying } = useProductMeta(product);
 
     // Add views to productView object where the keys are the product name and the values are the view component
     const productView = {
-        PAY_LATER_LONG_TERM: <LongTerm {...useContent(product)} contentBodyRef={contentBodyRef} />
+        PAY_LATER_LONG_TERM: <LongTerm {...useContent(product)} contentBodyRef={contentBodyRef} />,
+        PAY_LATER_SHORT_TERM: <ShortTerm {...useContent(product)} contentBodyRef={contentBodyRef} />
```

Comment:
```suggestion
        PAY_LATER_SHORT_TERM: <ShortTerm {...useContent(product)} {...useProductMeta(product)} contentBodyRef={contentBodyRef} />
```
If we pass in the product meta information into here, we can actually simplify the check inside of the Pi4's ```Content.jsx``` file.